catalog/docs/src/main/resources/_contents/integrating-contents.adoc

```diff
@@ -1194,18 +1194,18 @@ Otherwise, caching is canceled if the user cancels the product download.
 
```

Comment:
The QueryResponse on line 1151 has a white font color and is hard to read.

![screen shot 2016-03-01 at 12 42 33 pm](https://cloud.githubusercontent.com/assets/11355332/13439504/20246a48-dfab-11e5-80d7-f494f59b9601.png)